packages/contracts-bedrock/tasks/genesis-l2.ts

```diff
@@ -251,6 +369,42 @@ task('genesis-l2', 'create a genesis config')
       }
     }
 
+    const immutables = {
+      OptimismMintableERC20Factory: {
+        bridge: predeploys.L2StandardBridge,
+      },
+      GasPriceOracle: {
+        MAJOR_VERSION: toBytes32(0),
+        MINOR_VERSION: toBytes32(0),
+        PATCH_VERSION: toBytes32(1),
+      },
+      L1Block: {
+        MAJOR_VERSION: toBytes32(0),
+        MINOR_VERSION: toBytes32(0),
+        PATCH_VERSION: toBytes32(1),
+      },
+      L2CrossDomainMessenger: {
+        MAJOR_VERSION: toBytes32(0),
+        MINOR_VERSION: toBytes32(0),
+        PATCH_VERSION: toBytes32(1),
+      },
+      L2StandardBridge: {
+        MAJOR_VERSION: toBytes32(0),
+        MINOR_VERSION: toBytes32(0),
+        PATCH_VERSION: toBytes32(1),
+      },
+      L2ToL1MessagePasser: {
+        MAJOR_VERSION: toBytes32(0),
+        MINOR_VERSION: toBytes32(0),
+        PATCH_VERSION: toBytes32(1),
+      },
+      SequencerFeeVault: {
```

Comment:
Should this PR also make [this value](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/L2/SequencerFeeVault.sol#L24) and set it's value?

Reply:
I didn't want to do so because it was a contract implementation change and the audit is going on

Reply:
Actually, I don't think this would require a change to the solidity?